src/client/styles/components/_form-component.scss

```diff
@@ -0,0 +1,13 @@
+.message {
+    border-top: 2px solid #ffb81c;
+    background-color: #fff8e8;
```

Comment:
It's good practice to use variables for colours. Maybe see if the colours from the [NHS colour palette](https://github.com/nhsuk/frontend/blob/master/src/scss/environment/settings/_colours.scss) can be applied first and if not, create a local colours file for this project in a similar location (`/environment/settings/_colours.scss`).

For reference, this setup is based on the [inverted triangle](https://www.xfive.co/blog/itcss-scalable-maintainable-css-architecture/) approach for structuring CSS. Might be worth a read which will help understand the structure of the files.